packages/nextjs/app/builders/0x2EdDb96A06A97DafD28805e780a45661639aDe43/page.tsx

```diff
@@ -0,0 +1,60 @@
+import Image from "next/image";
+import Link from "next/link";
+import Greeter from "./components/Greater";
+import SecretInfo from "./components/SecretInfo";
+import { FaGithub } from "react-icons/fa";
+import { FaLinkedin } from "react-icons/fa";
+
+export default function DiegoPortfolio() {
```

Comment:
You might want to consider typing your page as NextPage since we are using TypeScript, as this ensures better integration and adds clarity for future maintainers by indicating that the component is a Next.js page.
```suggestion
const DiegoPortfolio: NextPage = () => {
```

Reply:
Perfect. Done.